lib/grinder_tools.dart

```diff
@@ -155,13 +160,15 @@ class Pub {
    * even if the pubspec.lock file is up-to-date with respect to the
    * pubspec.yaml file.
    */
-  static void get({bool force: false, String workingDirectory}) {
+  static String get({bool force: false, String workingDirectory}) {
```

Comment:
I don't have a strong preference here. `String` is consistent. For several of the operations the caller really won't care about the result. For those I would prefer `String` or `void` over `bool`.